plugins/basic/nexus-repository-maven/src/main/java/org/sonatype/nexus/repository/maven/internal/maven2/metadata/MetadataRebuilder.java

```diff
@@ -59,10 +58,9 @@
 import com.orientechnologies.orient.core.metadata.schema.OType;
 import com.orientechnologies.orient.core.record.impl.ODocument;
 import com.orientechnologies.orient.core.sql.query.OSQLAsynchQuery;
-import com.orientechnologies.orient.core.tx.OTransaction.TXTYPE;
-import org.w3c.dom.Document;
-import org.w3c.dom.NodeList;
-import org.xml.sax.SAXException;
+import org.codehaus.plexus.util.xml.Xpp3Dom;
```

Comment:
As Tamas mentions in the PR description `Xpp3DomBuilder` is more resilient to malformed Maven poms and better matches what Maven itself uses, which avoids inconsistencies. Note that since this is a pure library it doesn't involve any use of Plexus injection and is limited to this module.